AutomaticQC/imosTiltVelocitySetQC.m

```diff
@@ -241,27 +267,55 @@
 path = fullfile(path, 'AutomaticQC');
 
 fid = -1;
-params = [];
 try
   fid = fopen([path filesep 'imosTiltVelocitySetQC.txt'], 'rt');
   if fid == -1, return; end
-  
   params = textscan(fid, '%s%f%f%d8%d8', 'delimiter', ',', 'commentStyle', '%');
   fclose(fid);
 catch e
   if fid ~= -1, fclose(fid); end
   rethrow(e);
 end
 
-% look for an instrument match
-for i=1:length(params{1})
-    if ~isempty(strfind(lower(instrument), lower(params{1}{i})))
-        firstTiltThreshold  = params{2}(i);
-        secondTiltThreshold = params{3}(i);
-        firstTiltflag  = params{4}(i);
-        secondTiltflag = params{5}(i);
-        return;
-    end
+names = params{1};
+nlen = length(names);
+
+%first try to match first name
+for k=1:nlen
+  if strcmpi(instrument,names{k})
+    matchedName = names{k};
+    firstTiltThreshold = params{2}(k);
+    secondTiltThreshold = params{3}(k);
+    firstTiltflag = params{4}(k);
+    secondTiltflag = params{5}(k);
+    return
+  end
+end
+
+%then try to match the instrument name to any of options
+submatch = contains(names,instrument,'IgnoreCase',true);
+if any(submatch)
+  k = find(submatch,1);
+  matchedName = names{k};
+  firstTiltThreshold = params{2}(k);
+  secondTiltThreshold = params{3}(k);
+  firstTiltflag = params{4}(k);
+  secondTiltflag = params{5}(k);
+  return
 end
 
-end
\ No newline at end of file
+%finally, try to match specific options on the table as substrings in the instrument
+%name - useful when loading from netcdfs
+for k=1:nlen
+  invmatch = contains(instrument,names{k},'IgnoreCase',true);
```

Comment:
further below, I see there are both long ranger and longranger (not from your PR though).
I have the feeling IgnoreCase should be true, but also a "IgnoreWhitespaces' or something like this, some sort of fuzzysearchfinder

Reply:
Haha, we follow the same line of thought. I even did implement a Levenshtein distance for fuzzy matching and tested it, but it was not backward compatible with current entries.

Reply:
ah thats a shame!